src/script/auth/page/Root.tsx

```diff
@@ -61,6 +57,28 @@ import * as AuthSelector from '../module/selector/AuthSelector';
 import * as LanguageSelector from '../module/selector/LanguageSelector';
 import {ROUTE} from '../route';
 
+/**
+ *  Lazy loading components for routes, which can only be accessed by activated feature flags
+ */
+const E2EIdentityOidcRedirect = loadable(() => import('./E2EIdentityOidcRedirect'), {
```

Comment:
im getting a cant resolve error here when running locally

Reply:
also just curious why using loadable for lazy loading instead of react.lazy? SSR?